phpunit.xml.dist

```diff
@@ -6,7 +6,11 @@
          executionOrder="random"
          failOnWarning="true"
          failOnRisky="true"
+         failOnDeprecation="true"
          requireCoverageMetadata="true"
+         displayDetailsOnPhpunitDeprecations="true"
```

Comment:
Just show details in CI so I as developer could see the reason why build is failed